app/models/feature.rb

```diff
@@ -1,12 +1,12 @@
 class Feature
-  def self.enabled?(feature_name)
+  def self.enabled?(feature_name, user: nil)
```

Comment:
Please remove the defaulted parameter here. It needs to be required so it doesn't slip through. All components using `Feature` will need to pass in a user.